packages/select/src/vaadin-lit-select.js

```diff
@@ -41,12 +41,17 @@ class Select extends SelectBaseMixin(ElementMixin(ThemableMixin(PolylitMixin(Lit
       inputFieldContainer,
       screenReaderOnly,
       css`
-        :host {
-          position: relative;
```

Comment:
Please restore `position: relative` since it's needed for `sr-only` element (which uses `position: absolute`) and there is a test for it. See also https://github.com/vaadin/web-components/pull/6388.